master/networking/vxlan-ipip.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Connecting workloads that you do not control
+---
+
+### Big Picture
+
+Enable inter workload communication across networks that are not workload aware.
```

Comment:
Should "inter workload" be hyphenated?

Reply:
I thought the same, but it appears that without is what comes up in search.